system/setup.py

```diff
@@ -139,6 +171,9 @@ def main():
     global module
     module = AnsibleModule(
         argument_spec = dict(
+            gather_subset=dict(default="all", required=False),
```

Comment:
This should probably be a list type so that the module parameter validation and conversion makes it a list.  It should turn a comma separated string into a list for us.